substratest/factory.py

```diff
@@ -14,7 +14,7 @@
 
 DEFAULT_DATA_SAMPLE_FILENAME = 'data.csv'
 
-DEFAULT_SUBSTRATOOLS_VERSION = '0.6.0'
+DEFAULT_SUBSTRATOOLS_VERSION = '0.6.1'
```

Comment:
```suggestion
DEFAULT_SUBSTRATOOLS_VERSION = '0.6.1-minimal'
```

@Esadruhn I merged https://github.com/SubstraFoundation/substra-tests/pull/118 to use the minimal docker image, so you should be good to use the minimal version here too.

The `0.6.1-minimal` version was automatically built [on docker hub](https://hub.docker.com/layers/substrafoundation/substra-tools/0.6.1-minimal/images/sha256-2d061db2938587a05df3a4f66074d03f44f4fc74851a7f1d0ff645671ca0fbe8?context=repo)